pyrk/data/precursors.py

```diff
@@ -86,8 +86,8 @@ def _get_Lambda(self, nuc, e):
         Lambda_dict = {}
         Lambda_dict["u235"] = {}
         Lambda_dict["pu239"] = {}
-        Lambda_dict["sfr"] = {}
         Lambda_dict["u235"]["thermal"] = 1.08e-5
+        Lambda_dict["sfr"] = {}
```

Comment:
this is an odd change, as it simply moves the SFR line between to U235 lines. This doesn't affect the code behavior, but does affect readability. I won't require that you change it, but since it is an entirely unneccessary change, I would prefer that it be put back as it was before.